x-pack/plugins/secops/public/components/page/navigation/index.tsx

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import {
+  EuiButton,
+  EuiSpacer,
+  // @ts-ignore
+  EuiTab,
+  // @ts-ignore
+  EuiTabs,
+} from '@elastic/eui';
+import * as React from 'react';
+import { pure } from 'recompose';
+import styled from 'styled-components';
+
+import { getHostsUrl, getNetworkUrl, getOverviewUrl } from '../../link_to';
+
+interface NavTab {
+  id: string;
+  name: string;
+  href: string;
+  disabled: boolean;
+}
+
+const navTabs: NavTab[] = [
+  {
+    id: 'overview',
+    name: 'Overview',
+    href: getOverviewUrl(),
+    disabled: false,
+  },
+  {
+    id: 'hosts',
+    name: 'Hosts',
+    href: getHostsUrl(),
+    disabled: false,
+  },
+  {
+    id: 'network',
+    name: 'Network',
+    href: getNetworkUrl(),
+    disabled: false,
+  },
+];
+
+export const Navigation = pure(() => (
+  <div>
+    <EuiTabs>{renderTabs()}</EuiTabs>
+    <AddData />
+    <EuiSpacer />
+  </div>
+));
+
+const renderTabs = () =>
+  navTabs.map((tab: NavTab) => (
+    <EuiTab
+      onClick={() => window.location.assign(tab.href)}
+      isSelected={window.location.hash.includes(tab.id)}
+      disabled={tab.disabled}
+      key={`navigation-${tab.id}`}
+    >
+      {tab.name}
+    </EuiTab>
+  ));
+
+const AddData: React.SFC = () => (
+  <div
+    style={{
+      margin: '0 5px',
```

Comment:
I would move this to be a styled component and get rid of the `style={{`

Reply:
> Checked it out and played with it a bit. I don't know if we want to change the underlines to move to the new locations as they are clicked with this PR but they do not move when clicked at the moment.
> 
> For example, when I click "network" I get "hello network" but the underline does not move with it:
> <img alt="screen shot 2018-11-05 at 8 20 13 pm" width="1036" src="https://user-images.githubusercontent.com/1151048/48041044-aa841080-e138-11e8-84fd-af81353c76c2.png">
> 
> If not on this PR, I would suggest a bug ticket for our backlog.
> 
> Looking at the code it is checking for it:
> 
> ```
>     <EuiTab
>       onClick={() => window.location.assign(tab.href)}
>       isSelected={window.location.hash.includes(tab.id)}
>       disabled={tab.disabled}
>       key={`navigation-${tab.id}`}
>     >
>       {tab.name}
>     </EuiTab>
> ```
> But it's not showing up operating correctly?

Good Catch!!! Focusing too much on bad coding, it will make you miss the real stuff :)